### ISSUE TITLE
make it easy to enable StackDriver if on GKE

### ISSUE THREAD

**Issue by jstrachan** [jenkins-x/jx#942]
it'd be nice if we installed jenkins X on GKE to detect the GKE provider type and either default or prompt the user to enable stack driver on all apps

**Comment by garethjevans** [comment]
Stackdriver should be enabled by default on all GKE clusters (there is a flag to disable it or use another service) - providing the account is activated.

Or are you thinking of something inside each pod to collect metrics/logs?

**Comment by rawlingsj** [comment]
I think that should be enough provided apps log to standard out?

**Comment by garethjevans** [comment]
Will double check and let you know

**Comment by rawlingsj** [comment]
Ok cool, I'm pretty sure that's the case as when I've manually enabled it before we get the logs although most apps dont tend to log in JSON so we loose some of stackdriver's power.

BTW I just noticed that there's stackdriver monitoring with GKE Kubernetes 1.10.2 https://cloud.google.com/monitoring/kubernetes-engine.  No idea what that looks like or how it compares with prometheus but figured it's worth mentioning and might be worth looking at.
